src/static/css/nav-primary.less

```diff
@@ -0,0 +1,459 @@
+                        tabindex="0">
+                        About
+                    </a>
+                    <div class="sub-nav_wrapper" aria-expanded="false">
+                        <div class="sub-nav_inner-wrapper wrapper">
+                            <section class="sub-nav_menu">
+                                <h4 class="sub-nav_title">
+                                    About Overview
+                                </h4>
+                                <hr class="content-line content-line__nav">
+                                <ul class="sub-nav_menu-list
+                                           list__unstyled
+                                           list-expanding
+                                           list-expanding_child-list">
+                                    <li class="list_item list-expanding_list-item">
+                                        <a class="list_link list_link" href="/the-bureau/">
+                                            The Bureau
+                                        </a>
+                                    </li>
+                                </ul>
+                            </section>
+                            <aside class="sub-nav_media">
+                                <a href="/the-bureau/">
+                                    <img src="../../static/img/nav-about-us-video.png"
+                                         alt="About Us video image">
+                                    <h4 class="short-desc u-mt15">
+                                        The CFPB: Working for you.
+                                    </h4>
+                                </a>
+                                <div class="short-desc">
+                                    This short video covers what the CFPB is and
+                                    how we are working for American consumers.
+                                </div>
+                            </aside>
+                        </div>
+                    </div>
+                </li>
+            </ul>
+        </nav>
+      notes:
+        -
+  tags:
+    - cfgov-nav-primary
+*/
+
+.primary-nav .list_link {
+    .u-link__colors(@black, @green);
+    .webfont-regular();
+}
+
+
+/* Sliding nav for portrait tablets and smaller
+   ========================================================================== */
+
+.respond-to-max(@tablet-max, {
+
+    html,
+    body {
+        height: 100%;
+    }
+
+    .u-show-on-desktop {
+        display: none;
+    }
+
+    // Remove extra padding added by .wrapper__match-content so the nav button
+    // is flush left with the screen.
+    .primary-nav_header-space.wrapper__match-content {
+        padding-left: 0;
+        padding-right: 0;
+    }
+
+    .sub-nav_menu-list {
+        width: 100%;
+
+    }
+
+    .sub-nav_menu-list .list_link {
+        // Override standard visited link color.
+        .u-link__colors(@pacific, @pacific, @pacific-50, @pacific, @navy);
```

Comment:
If we more directly targeted the changes on [L:69](https://github.com/cfpb/cfgov-refresh/pull/685/files#diff-326cef213344c8a38e386ce6019ac100R69) could we remove this since these are the defaults?